roles/servicetelemetry/tasks/component_grafana.yml

```diff
@@ -14,10 +23,32 @@
       '{{ grafana_manifest }}'
   when: servicetelemetry_vars.graphing.enabled
 
-# Lookup existing datasources
-- block:
+- when: servicetelemetry_vars.graphing.enabled
+  block:
+  - when: servicetelemetry_vars.backends.metrics.prometheus.enabled
+    block:
+    - name: Retrieve configmap for OAUTH CA certs
+      k8s_info:
+        api_version: v1
+        kind: ConfigMap
+        name: serving-certs-ca-bundle
+        namespace: '{{ meta.namespace }}'
+      register: serving_certs_ca
 
-  - block:
+    - name: Retrieve prometheus secret
+      k8s_info:
+        api_version: v1
+        kind: Secret
+        namespace: '{{ meta.namespace }}'
+        name: '{{ meta.name }}-prometheus-htpasswd'
+      register: prometheus_secret
+
+    - name: Decode prometheus password
+      set_fact:
+        prom_basicauth_passwd: '{{ prometheus_secret.resources[0].data.password | b64decode }}'
```

Comment:
Curiously I'm not seeing anything about this particular task in the log, even though I do see the ones above and below. I'm going to add no_log in a few places anyways just in case something changes or a different verbosity setting would cause trouble.

Reply:
Yea I think the default verbosity is 2. You can add the `ANSIBLE_VERBOSITY` env var to the CSV (or `operator.yaml` if generating the CSV) to set the verbosity level.

https://docs.ansible.com/ansible/latest/reference_appendices/config.html#envvar-ANSIBLE_VERBOSITY

It's possible setting the verbosity to a higher level (like 4) could cause that information to be displayed. I don't think the risk is all that high because if you can read the logs you're already an administrator basically, but better safe than sorry I suppose.